api/securitytest/gitleaks.go

```diff
@@ -42,6 +42,23 @@ func analyseGitleaks(gitleaksScan *SecTestScanInfo) error {
 		return nil
 	}
 
+	// if gitleaks timeout, a warning will be generated as a low vuln
```

Comment:
Would you mind changing this comment to `nohusky vuln`? Reading the rest of the changes, I believe that a warning will be added as a `nohusky vuln`.

Reply:
Sure, @Krlier! Good catch. I have fixed it at a3e16ab6d4e98610dd7889acbf4016a30a48593f.